docs/miscellaneous.rst

```diff
@@ -87,10 +87,14 @@ Solidity always places new objects at the free memory pointer and memory is neve
 Layout of Call Data
 *******************
 
-When a Solidity contract is deployed and when it is called from an
-account, the input data is assumed to be in the format in :ref:`the ABI
-specification <ABI>`. The ABI specification requires arguments to be padded to multiples of 32
-bytes.  The internal function calls use a different convention.
+The input data for a function call is assumed to be in the format in the :ref:`ABI
```

Comment:
Would `...assumed to be in the :ref: ABI specification <ABI> format.` be better?

Reply:
.. in the format defined by the ABI specification?

Reply:
Sounds good!